x-pack/solutions/security/plugins/security_solution/public/flyout/ui_tests/parallel_tests/global.setup.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { globalSetupHook } from '@kbn/scout';
```

Comment:
(Nit) minor difference between the Observability solution's file name `global_setup.ts` ([file](https://github.com/elastic/kibana/blob/main/x-pack/solutions/observability/plugins/apm/ui_tests/parallel_tests/global_setup.ts)) and this file name `global.setup.ts`. Keeping different file name is fine by now (though it may make navigation between files of different packages a bit more difficult), but I wanted to bring this to your attention.